src/taipy/templates/multi-page-gui/{{cookiecutter.application_name}}/pages/page_1/page_1.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from taipy.gui import Markdown
+
```

Comment:
@FabienLelaquais @FlorianJacta Don't you think we can add some comments in the file to explain the page scoping or a link to the documentation page?

Reply:
Good idea.

I would certainly replace the copyright header (which is useless) with an introduction of what was generated,
and what can or needs to be customized.
Pointing to the doc does make sense (https://docs.taipy.io/en/latest/manuals/...)

Reply:
I add some intro string to the generated python files. Please take a look.

If it's ok, I will update the default template as well.